src/main/java/com/example/SteDziPanki/model/Country.java

```diff
@@ -0,0 +1,48 @@
+package com.example.SteDziPanki.model;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+public class Country {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+    private String name;
+
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL, mappedBy = "country" )
```

Comment:
Eager tylko tam gdzie jest to wymagane, czyli prawie nigdzie, Lazy jest tam podstawowo z jakiego powodu prawda?